mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentFallbackTest.java

```diff
@@ -64,6 +65,7 @@
 
     @BeforeClass
     public static void setUpClass() {
+        DockerCassandraSingleton.restart();
```

Comment:
why?

Reply:
I don't get a clue. But these specific class needs a restart to pass.

It may show some underlying problems. I will clearly open a JIRA about these test failures once the issue is merged.

Reply:
or maybe we can fix it before merging it